cpp/src/io/orc/stripe_init.cu

```diff
@@ -32,14 +32,12 @@ extern "C" __global__ void __launch_bounds__(128, 8) gpuParseCompressedStripeDat
 {
   __shared__ compressed_stream_s strm_g[4];
 
-  volatile compressed_stream_s *const s = &strm_g[threadIdx.x >> 5];
-  int strm_id                           = blockIdx.x * 4 + (threadIdx.x >> 5);
-  int t                                 = threadIdx.x & 0x1f;
+  compressed_stream_s *const s = &strm_g[threadIdx.x >> 5];
+  int strm_id                  = blockIdx.x * 4 + (threadIdx.x >> 5);
+  int t                        = threadIdx.x & 0x1f;
```

Comment:
This means `t` is lane id. Is it?

Reply:
Yes, it is lane id. I will update it with appropriate name.

Reply:
The code below seems to use it interchangeably. Be careful when updating.